src/components/03-components/03-buttons/01-button/button.config.js

```diff
@@ -0,0 +1,89 @@
+const _ = require("lodash");
+
+const variantNames = [
+  "default",
+  "secondary",
+  "base",
+  "danger",
+  "outline",
+  "outline-inverse",
+  "accent",
+  "white"
+];
+function variants() {
+  let variants = [];
+  let len = variantNames.length;
+
+  for (let i = 0; i < len; i++) {
+    let vname = variantNames[i];
```

Comment:
repeated variant sections should be added in main context area. Variant context data is only for places the variant data differs from main context area so in this case everything besides modifiers, sModifiers, classes, and disabled where present